.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
```

Comment:
We have a file named feature_request.md (underscore) and feature-request.md (hyphenated).  Do we really need both??